$NetBSD: patch-configure,v 1.8 2023/01/01 19:19:26 gutteridge Exp $

* Add NetBSD backend detection.

* Allow Polkit to be disabled in the build while still being present in the
underlying system.

--- configure.orig	2022-12-15 09:27:27.000000000 +0000
+++ configure
@@ -18012,7 +18012,7 @@ printf %s "checking for optional package
 printf "%s\n" "disabled" >&6; }
   fi
 
-   if test x"$POLKIT_FOUND" = x"yes"; then
+if test "${disable_polkit+set}" != set && test x"$POLKIT_FOUND" = x"yes"; then
   HAVE_POLKIT_TRUE=
   HAVE_POLKIT_FALSE='#'
 else
@@ -18307,6 +18307,8 @@ if test x$with_backend = x; then
     with_backend=freebsd ;; #(
   *-*dragonfly*) :
     with_backend=freebsd ;; #(
+  *-*netbsd*) :
+    with_backend=netbsd ;; #(
   *-openbsd*) :
     with_backend=openbsd ;; #(
   *) :
@@ -18334,6 +18336,11 @@ if test x$with_backend = xopenbsd; then
 printf "%s\n" "#define BACKEND_TYPE_OPENBSD 1" >>confdefs.h
 
 fi
+if test x$with_backend = xnetbsd; then
+
+printf "%s\n" "#define BACKEND_TYPE_NETBSD 1" >>confdefs.h
+
+fi
 
 ac_config_files="$ac_config_files Makefile doc/Makefile icons/Makefile icons/16x16/Makefile icons/24x24/Makefile icons/24x24-actions/Makefile icons/32x32/Makefile icons/48x48-actions/Makefile icons/48x48/Makefile icons/128x128/Makefile icons/scalable/Makefile libxfsm/Makefile po/Makefile.in settings/Makefile scripts/Makefile scripts/xinitrc.in xfce4-session/Makefile xfce4-session-logout/Makefile xfsm-shutdown-helper/Makefile"
 
